docs/guides/environmental-variables.md

```diff
@@ -0,0 +1,57 @@
+---
+title: CLI
+---
+
+Environmental variables are key-value pairs that are stored in our operating systems. Many of them come by default, and we can also create our own. They are used to store information that is used by our operating system or by other programs. For example, the `PATH` variable stores a list of directories that the operating system should look in when we run a command in the terminal. The `HOME` variable stores the path to our home directory.
+
+The same can be used for building softwares aswell! Configuration values are often stored in environmental variables.
```

Comment:
Avoid `!` in documentation. Keep the tone neutral.

Reply:
hmmmm got that.